src/libs/ReportUtils.ts

```diff
@@ -5565,6 +5565,10 @@ function isDeprecatedGroupDM(report: OnyxEntry<Report>): boolean {
     );
 }
 
+function isParentGroupChat(report: OnyxEntry<Report>): boolean {
```

Comment:
This name is ambiguous because it sounds like we're asking "is the parent a group chat". I'd suggest something like `isTopLevelGroupChat()`.

Reply:
@marcaaron What do you think about `isRootGroupChat`?

Reply:
That's fine too.